util/mk-fipsmodule-cnf.pl

```diff
@@ -15,7 +15,7 @@
 my $ems_check = 1;
 my $no_short_mac = 1;
 my $drgb_no_trunc_dgst = 1;
-my $kdf_digest_check = 1;
+my $digest_check = 1;
```

Comment:
It probably doesnt matter, but I would have left them as 2 seperate items..

Reply:
Meh.  We want them all enabled by default.